packages/@webex/plugin-meetings/test/unit/spec/meetings/index.js

```diff
@@ -794,7 +795,7 @@ describe('plugin-meetings', () => {
               eventType: test1,
               locusUrl: url1,
             });
-            assert.callCount(webex.meetings.meetingCollection.getByKey, 4);
+            assert.callCount(webex.meetings.meetingCollection.getByKey, 5);
             assert.calledWith(webex.meetings.meetingCollection.getByKey, 'locusUrl', url1);
```

Comment:
issue: I think there should be a test here to check the new behavior of `getByKey(MEETINGNUMBER)`